lessons/beginners/comparisons/index.md

```diff
@@ -93,24 +93,23 @@ print(False)
 
 ## Podmínky
 
-{% if var('pyladies') -%}
-{% set rootname = 'pyladies' %}
-{%- else -%}
-{% set rootname = 'naucse-python' %}
-{%- endif %}
-
 Teď oprášíme program na výpočet obvodu a obsahu.
-Do nového souboru, třeba <code><span class="pythondir">~/{{ rootname }}</span>/02/if.py</code>,
-napiš následující program.
-Co se stane, když jako stranu zadáš záporné číslo?
-Dává výstup smysl?
+
+Otevři si v editoru nový soubor.
+V adresáři, kde máš soubory ke kurzům Pythonu, udělej adresář s číslem lekce
+(třeba `02`).
```

Comment:
Can we get the lesson numerical ID programmatically? Only if it's easy.

Reply:
It's not particularly hard, but a bit lower priority for me. Filed as https://github.com/pyvec/naucse.python.cz/issues/444.